### PR TITLE
Update foreman_expire_hosts to 6.0.1

### PR DESCRIPTION
**Pull request by timogoebel**
(cherry picked from commit 1ff39ef1431700b56fbbf6fc766dfd471756f9ae)

### CODE REVIEW

**Comment by evgeni** [comment]
When updating to 6.0.0 in rpm/develop (32d9126f553898d2a60512c6f05ebb73580172b2) you bumped foreman_min_version to 1.18.0, this is not CP'ed. intentionally? Generally I'd prefer if the history in develop and in stable branches is "identical" (i.e. not skipping the 6.0.0 commit)

**Comment by timogoebel** [comment]
@evgeni: Thanks for the review. I added both commits to the PR.

File: packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec [hunk 1]
```diff
@@ -116,6 +116,12 @@ install -Dp -m0644 %{SOURCE1} %{buildroot}%{?scl:%_root_sysconfdir}%{!?scl:%_sys
 exit 0
 
 %changelog
+* Fri Mar 08 2019 Timo Goebel <mail@timogoebel.name> - 6.0.0-1
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec]: you got the changelog history wrong ;)

Comment (timogoebel) [reply packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec]: Darn, that could have been avoided.

🚂 

Fixed.

**Pull request merged**